public/app/features/alerting/unified/components/rule-viewer/tabs/AlertVersionHistory.tsx

```diff
@@ -23,6 +25,7 @@ import { VersionHistoryTable } from './version-history/VersionHistoryTable';
 const { useGetAlertVersionHistoryQuery } = alertRuleApi;
 
 interface AlertVersionHistoryProps {
+  combinedRule: CombinedRule;
   ruleUid: string;
 }
```

Comment:
As you pass `ruleUid` already, maybe the `AlertVersionHistory` component could fetch rule details using `getAlertRule` endpoint